src/components/chat/comment-card.tsx

```diff
@@ -46,13 +50,14 @@ export const CommentCard: React.FC<IProps> = ({ activeNavTab, user, postedCommen
   };
 
   const renderThreadHeader = () => {
-    const teacherInitial = user?.name.charAt(0);
     return (
       <div className="comment-card-header comment-select" data-testid="comment-card-header">
-        {postedComments && postedComments.length < 1
-          ? <DocumentCommentIcon className="new-thread-header-icon" data-testid="document-comment-icon"/>
-          : <div className="initial" data-testid="teacher-initial">{teacherInitial}</div>
-        }
+        <div className="comment-card-header-icon" data-testid="comment-card-header-icon">
+          { (!documentKey || !focusTileId) && <OpenWorkspaceIcon/>}
+          { documentKey && focusTileId
+             &&  <ToolIconComponent documentKey={documentKey} tileId={focusTileId}/>
```

Comment:
Just a nit, but there is an extra space after the `&&` that we could remove:
`&& <ToolIconComponent documentKey={documentKey} tileId={focusTileId}/>`

Reply:
Changed.